### ISSUE TITLE
Supporting Https for publishing data

### ISSUE THREAD

**Issue by nghia1986** [fastly/pushpin#47670]
Pushpin supports https for the proxy process, however there is no option on pushin-handler. When push data, It's only supporting http. Can you support https? Or is there any secure socket that I can use?

**Comment by jkarneges** [comment]
That's correct. The control ports are meant to be used from the server-side within a private network. If you need additional security (and I assume you'd want not just https but also authentication), you'll need to add your own layers.
